docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'tools.sphinxext.c7n_schema',
```

Comment:
In my development environment the root folder (where `c7n` and `tools` are) is in sys.path, but the `tools` subdirectory is not.  So `make sphinx` will fail if I just try to import `sphinxext`.  Ditto, I can't import `c7n_salactus`.

Is your environment set up differently?

Of course, I could manually push the tools directory into sys.path in the sphinx `conf.py` file, but I suspect that's not what you are going for.